ext/opentelemetry-ext-django/tests/test_middleware.py

```diff
@@ -111,3 +114,25 @@ def test_error(self):
             span.attributes["http.url"], "http://testserver/error/"
         )
         self.assertEqual(span.attributes["http.scheme"], "http")
+
+    @patch(
+        "opentelemetry.ext.django.middleware._DjangoMiddleware._excluded_urls",
+        ExcludeList(["http://testserver/excluded_arg/123", "excluded_noarg"]),
+    )
+    def test_exclude_lists(self):
+        client = Client()
+        client.get("/excluded_arg/123")
```

Comment:
Should add tests where the blocked url is a (proper) prefix, suffix and substring.